CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contribution Guidelines
+
+Thanks for your interest in contributing to PyCon India Blog. Your valuable contribution is much appreciated.
+
+The following is a set of guidelines for contributing to PyCon India Blog.
+
+## How Can I Help?
+
+Start looking through the issues and watch out for any issues with the `bug` or `help wanted` labels. Issues marked with these labels haven't been assigned to anyone yet and we would love for you to work on them.
+
+## Request a feature
+
+If you think there is a cool feature you would like to add please create an issue for that and we will assign that to you if the feature looks good to add.
+
+## Reporting bugs
+
+If you find any bugs, it would be great if you would let us know so we can look into it. You can do so by creating an issue. If you don't know how to create an issue, please follow [these instructions](https://help.github.com/articles/creating-an-issue/). Screenshots and notes for how you encountered the bug are really helpful when reporting bugs!
+
+## Creating Pull Requests
+
+After you've created a patch that fixes an issue, please submit a pull request for it. Mention **@pythonindia/pycon-2018-team** to get the Pull Request reviewed. If you're not sure how to create a pull request, please follow [these instructions](https://help.github.com/articles/creating-a-pull-request/). In general we'd like to ask you to please target the master branch with your pull requests.
```

Comment:
Don't need to mention `@pythonindia/pycon-2018-team` every time, the moderators will look into the issue/pr when they can. Also, removing this would help the guideline to be more generic towards coming years.

Reply:
updated